jdaviz/core/tools.py

```diff
@@ -249,3 +249,44 @@ class SidebarShortcutCompass(_BaseSidebarShortcut):
     tool_id = 'jdaviz:sidebar_compass'
     action_text = 'Compass'
     tool_tip = 'Open compass plugin in sidebar'
+
+
+@viewer_tool
+class SinglePixelRegion(CheckableTool):
+
+    icon = os.path.join(ICON_DIR, 'select_single_pixel.svg')
+    tool_id = 'jdaviz:singlepixelregion'
+    action_text = 'Create single-pixel spatial region'
+    tool_tip = 'Click on the viewer to create single-pixel spatial region'
```

Comment:
Should this tooltip be closer to that of other subset tools which all say "Define a ___ region of interest" (although not sure how well that works with "pixel)?

Reply:
I updated the tooltip. Better?